totalRP3/Locales/Locale.lua

```diff
@@ -624,6 +624,7 @@ Possible status:
 	CO_LOCATION_DISABLE_CLASSIC_PVP_TT = "You will not respond to location requests from other players when you are flagged for PvP.\n\nThis option is particularly useful to avoid abuses of the location system to track you.",
 	CO_SANITIZER = "Sanitize incoming profiles",
 	CO_SANITIZER_TT = "Remove escaped sequences in tooltip fields from incoming profiles when TRP doesn't allow it (color, images ...).",
+	CO_DATE_FORMAT = "Date Format",
```

Comment:
There's a header section further down the Locale script for adding new localization strings - it should go underneath that section.